Makefile

```diff
@@ -43,7 +43,7 @@ black:
 	black setup.py samcli tests
 
 black-check:
-	black --check setup.py samcli tests
+	black setup.py samcli tests --check
```

Comment:
this change was not intentional. I don't want to change it back because pr checking is forever 😢

Reply:
If we're being pedantic the order _is_ `black [OPTIONS] [SRC]...`... I don't think `black` cares but other tools do.

Reply:
please don't ask me to change this tiny thing back. It will be another hour running the PR check 🥺

Reply:
Haha I didn't revoke the approval!

Reply:
since jacob is not likely to approve it today, let me just fix it